### ISSUE TITLE
Solved:Seems like GorillaPlayer has a problem with nested Namespaces in xmlns

### ISSUE THREAD

**Issue by escamoteur** [UXDivers/Gorilla-Player-Support-And-Issues#56]
Hi,

new Problem. As SampleData.json does not work I tried to define a constant Item source:

```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:TBGUI="clr-namespace:TBGUI;assembly=TBGUI"
             xmlns:local="clr-namespace:QuoteMyDay.Model;assembly=QuoteMyDay"
             x:Class="QuoteMyDay.DayPage"
              BackgroundColor="#21f3e3">
          
  <ContentPage.Content>

    <TBGUI:ListView x:Name="ItemList" HasUnevenRows="True">
      <ListView.ItemsSource>
          <x:Array Type="{x:Type local:MyDay}">
            <local:MyDay Quote="Test qote" DateAsString="31.12.2001" />
         </x:Array>
      </ListView.ItemsSource>

```

Unfortunately I get the Error in Gorilla SDK-Player as in attached Screenshot
![image](https://cloud.githubusercontent.com/assets/16635729/13614319/3976340c-e570-11e5-868e-1aa93200e72d.png)

**Comment by escamoteur** [comment]
I now found out, it had nothing to do with the namespace. The problem was that I even had to add my main App PCL assembly to Gorilla.json to access a class defined in it.

**Comment by Umar3x** [comment]
Hello @escamoteur , could you please share your gorilla .json ? I got the same error with a CustomMarker using syncfusion maps, tried to add a gorilla.json but the issue still there. Did you register your custom class in the obj config too ?

**Comment by LeoHere** [comment]
@Umar3x this issue is very old. You don't need to specify your known assemblies in the `gorilla.json` any more. See [this](https://github.com/UXDivers/Gorilla-Player-Support/wiki/Gorilla-SDK). 

For the next version of Gorilla we are working on a AutoConfig feature that remove the need for this configuration. If you want to test it write me to l r o d r i g u e z at uxdivers.com and i can share it with you.

**Comment by Umar3x** [comment]
Hello @LeoHere 

Thanks for your fast answer. Yep I read that, but as I still got an error but not on the page where I use this custom class. I got a tabbedPage in which I got that and it displays me that Type local:CustomMarker not found in xmlns:clr-namespace:MyProject.UIComponents 

```
 <pages:DetailPage ></pages:DetailPage>
    <pages:ChooseProfilPage></pages:ChooseProfilPage>
```

If I preview only the DetailPage alone, its working. 

In the detail page : 

` xmlns:local="clr-namespace:MyProject.UIComponents"`
